apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -245,14 +245,14 @@ export const getUnitNameRender = <T extends Record<string, unknown>>(intl, text:
     return getTableCellRenderer({ search, extraTitle })(unitName)
 }
 
-export const getDateRender = (intl, search?: FilterValue | string) => {
+export const getDateRender = (intl, search?: FilterValue | string, newLinePostfix = true) => {
```

Comment:
I think something like this can be more expandable
```suggestion
export const getDateRender = (intl, search?: FilterValue | string, prefix = '\n') => {
```